opensearch/src/main/java/org/opensearch/sql/opensearch/request/OpenSearchQueryRequest.java

```diff
@@ -45,9 +59,21 @@ public class OpenSearchQueryRequest implements OpenSearchRequest {
   /** List of includes expected in the response. */
   @EqualsAndHashCode.Exclude @ToString.Exclude private final List<String> includes;
 
+  @EqualsAndHashCode.Exclude private boolean needClean = true;
+
   /** Indicate the search already done. */
   private boolean searchDone = false;
 
+  private String pitId;
+
+  private OpenSearchClient client;
```

Comment:
I think we should not pass `client` to Request class. Similar as what you do in `cleanup`, following the current design, you probably need to create a PIT Request class with fields here and change Client class accordingly.

Reply:
Updated in the latest revision